dependencies/pom.xml

```diff
@@ -879,6 +882,12 @@
                 <version>${rocketmq-version}</version>
             </dependency>
 
+            <!-- Fury Serialize -->
```

Comment:
Does it need to be declared in the seata-all pom?

Reply:
This is not the pom.xml of the seata-all module, but the seata-dependencies module.